lib/controllers/spree/stripe/intents_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Spree
+  module Stripe
+    class IntentsController < ::ActionController::API
+      def handle_response
+        @order = Spree::Order.find_by!(number: params['order'])
```

Comment:
@chhga can we pass order token here and do a lookup by a token? it will be safer